autobahn/src/main/java/io/crossbar/autobahn/wamp/Session.java

```diff
@@ -170,7 +170,11 @@ public void onMessage(IMessage message) {
                 CallRequest request = mCallRequests.getOrDefault(msg.request, null);
                 if (request != null) {
                     mCallRequests.remove(msg.request);
-                    request.onReply.complete(new CallResult(msg.args, msg.kwargs));
+                    if (request.resultType == null) {
+                        request.onReply.complete(new CallResult(msg.args, msg.kwargs));
+                    } else {
+                        // XXXXX - Use serializer here ?
```

Comment:
@oberstet One option would be to pass in the serializer to onMessage() handler method, giving the Session access to the serializer.